.eslintrc.cjs

```diff
@@ -11,6 +11,27 @@ module.exports = {
 	rules: {
 		// add rules... or dont...
 		'no-debugger': 'error',
-		'@typescript-eslint/ban-ts-comment': 'off'
+		'@typescript-eslint/ban-ts-comment': 'off',
+		'@typescript-eslint/no-explicit-any': 'off',
+		'@typescript-eslint/no-non-null-assertion': 'off',
+		'@typescript-eslint/no-non-null-asserted-optional-chain': 'off',
```

Comment:
`no-non-null-asserted-optional-chain` - similar to above, this one should also be enabled for similar reasons.

Reply:
I agree but I think this should be saved for a separate task, probably a big change

Reply:
I agree but I think this should be saved for a separate task, probably a big change

Reply:
Yes can you please make an issue for this?